pandasql/core.py

```diff
@@ -480,7 +501,7 @@ def __init__(self, source: DataFrame, dest: DataFrame, col: str, value):
             val = val._operation_as_str()
 
         new_column = '{} AS {}'.format(val, self.col)
-        columns = columns.insert(len(columns), new_column)
+        columns = columns.insert(col_index, new_column)
```

Comment:
A small test for checking column-order after updates would be good.

Reply:
added